pkg/storage/replica_proposal_buf.go

```diff
@@ -445,11 +445,18 @@ func (b *propBuf) FlushLockedWithRaftGroup(raftGroup *raft.RawNode) error {
 				continue
 			}
 
-			if err := raftGroup.ProposeConfChange(raftpb.ConfChange{
+			confChange := raftpb.ConfChange{
 				Type:    changeTypeInternalToRaft[crt.ChangeType],
```

Comment:
Does it make sense to keep `changeTypeInternalToRaft` around anymore, now that there's not a 1:1 mapping between `roachpb.ReplicaChangeType` and `raftpb.ConfChangeType`?

Reply:
Didn't see this until just now, but it's actually getting removed in #39485!